pacmacs-score.el

```diff
@@ -50,21 +50,23 @@
     (let ((inhibit-read-only t))
       (erase-buffer)
       (-> (pacmacs--read-score-table)
-          (pacmacs--sort-score-table)
           (pacmacs--render-score-table)))))
 
 (defun pacmacs--read-score-table ()
   (when (file-exists-p pacmacs--score-file-name)
-    (-> pacmacs--score-file-name
-        (f-read-text)
-        (read-from-string)
-        (car))))
+    (->> pacmacs--score-file-name
+         (f-read-text)
+         (read-from-string)
+         (car)
+         (pacmacs--sort-score-table))))
```

Comment:
What if someone modified the file and unordered the table? It may lead to an incorrect behavior, because we rely on the order in most cases. Previously we did this sort outside of the read function. The whole point was to move it inside.